helm/charts/infra/README.md

```diff
@@ -0,0 +1,70 @@
+# Infra Chart
+
+A Helm chart for Infra, an infrastructure access management tool for Kubernetes.
+
+The default deployment installs `infra-server` and accompanying services including a frontend service and a PostgreSQL database. The full list of configuration values can be found with `helm show values infrahq/infra`.
+
+## Upgrade
+
+The upgrade is a standard Helm upgrade. Run the commands below to update the Helm repository and perform the upgrade. Ensure the correct values are specified to avoid any unnecessary changes to the deployment. As in the previous step, these commands assume the Helm release is called `infra`, update as needed.
+
+```bash
+RELEASE_NAME=infra
+RELEASE_NAMESPACE=$(helm status $RELEASE_NAME -o json | jq -r .namespace)
+HELM_ARGS=.. # additional arguments to helm, e.g. --set or --values
+
+### helm repo update infrahq
+helm -n $RELEASE_NAMESPACE upgrade $RELEASE_NAME infrahq/infra $HELM_ARGS
+```
+
+Once upgraded, verify the deployment with `infra login` or the Infra UI. Once verified, the server persistent volume claim should be disabled, `server.persistence.enabled=false`, to fully remove the SQLite database and old database encryption key.
+
+## Rollback
+
+If the upgrade is unsuccessful, the deployment can be rolled back using Helm. The upgrade can be attempted again or you can contact us through GitHub for additional assistance.
+
+```bash
+RELEASE_NAME=infra
+RELEASE_NAMESPACE=$(helm status $RELEASE_NAME -o json | jq -r .namespace)
+
+helm -n $RELEASE_NAMESPACE rollback $RELEASE_NAME
+```
+
+### 0.14.4
+
+Infra chart version 0.20.0 introduces two backwards incompatible changes. The first is the migration to PostgreSQL as the default backend database from SQLite. The second is the migration of the database encryption key from within the server PVC to a Kubernetes secrets. This guide will detail the required steps to successfully migrate an Infra deploy to 0.20.0.
+
+#### Database Encryption Key
+
+This change requires manual migration before performing the upgrade. If your deployment uses a database encryption key stored outside of Kubernetes, i.e. if your Helm values defines `server.config.dbEncryptionKey`, you can ignore this section.
+
+The following commands creates a Kubernetes secret with the necessary labels and annotations to masquerade as a Helm-managed resource. It seeds this secrets with the current encryption key such that it’s available to Infra after upgrade. 
+
+These commands assume the Helm release is called `infra`. Update the `RELEASE_NAME` variable if necessary with the correct release name. 
```

Comment:
Does this get done before, or after the upgrade?

Reply:
These steps need to be run before the upgrade